internal/pkg/template/templates/task/cf.yml

```
@@ -69,8 +58,11 @@ Resources:
           Environment:{{range $name, $value := .EnvVars}}
           - Name: {{$name}}
             Value: {{$value}}{{end}}{{end}}
-          {{- if .Secrets}}
-          Secrets:{{range $name, $valueFrom := .Secrets}}
+          {{- if or .SSMParamSecrets .SecretsManagerSecrets}}
+          Secrets:{{range $name, $valueFrom := .SSMParamSecrets}}
+          - Name: {{$name}}
+            ValueFrom: {{$valueFrom}}{{end}}
+          {{- end}}{{- if .SecretsManagerSecrets }}{{range $name, $valueFrom := .SecretsManagerSecrets}}
```

Can we remove the first part? 
```suggestion
          {{range $name, $valueFrom := .SecretsManagerSecrets}}
```